.github/workflows/ci.yaml

```diff
@@ -17,19 +17,19 @@ env:
 
 jobs:
   build:
-    name: Build on ${{ matrix.target }}
+    name: Build on ${{ matrix.target.triple }}
     strategy:
       matrix:
-        include:
-          - target: x86_64-apple-darwin
+        target:
+          - triple: x86_64-apple-darwin
```

Comment:
Should we maybe rename `triple` to `arch` or something like that? I'm not exactly sure what `triple` should represent.

Reply:
"Target triple" is a term used by Rust - see https://doc.rust-lang.org/nightly/rustc/platform-support.html

> Targets are identified by their "target triple" which is the string to inform the compiler what kind of output should be produced.

Reply:
In that case I think we're okay with using the term `triple`. Thanks for providing the resource!